app/src/test/kotlin/com/intive/picover/auth/repository/AuthRepositoryTest.kt

```diff
@@ -137,14 +137,13 @@ class AuthRepositoryTest : ShouldSpec(
 
```

Comment:
was also thinking about inlining:

```
val referenceToUserAvatar: StorageReference = mockk {
			every { putFile(userPhoto) } returns mockk {
				every { isComplete } returns true
				every { exception } returns null
				every { isCanceled } returns false
				every { result } returns mockk()
			}
		}
```
with

```
val storageReference: StorageReference = mockk(relaxed = true) {
  every { child("user/$userUid") } returns referenceToUserAvatar
}
```

but i am not sure if referencing to, for example:

` storageReference.child("user/$userUid").downloadUrl.await()`

wouldn't be to complicated in case of getting to know what is going on here :D